tests/dummy/app/templates/docs/components/au-file-upload.md

```diff
@@ -0,0 +1,39 @@
+# File upload
```

Comment:
Since there is a webuniversum version maybe it makes sense to add an entry to the [migration page](https://appuniversum.github.io/ember-appuniversum/docs/outline/webuniversum) as well?